modules/ROOT/pages/dw-core-functions-unzip.adoc

```diff
@@ -52,8 +52,8 @@ unzip([ [0,"a"], [1,"b"], [2,"c"],[ 3,"d"] ])
 === Example
 
 This example unzips an array of arrays. Notice that the number of elements in
-the input arrays are not all the same. The function only creates as many full
-sub-arrays as it can, in this case just one.
+the input arrays is not all the same. The function creates only as many full
```

Comment:
tense mismatch

Reply:
"... the number of elements ... is " leaving as-is